client/dashboard/app/queries.ts

```diff
@@ -240,3 +244,41 @@ export function siteDefensiveModeMutation( siteSlug: string ) {
 		},
 	};
 }
+
+export function siteHasPurchasesCancelableQuery( siteId: string, userId: number ) {
+	return {
+		queryKey: [ 'site', siteId, 'purchases-cancelable' ],
```

Comment:
> This option can be used to transform or select a part of the data returned by the query function. It affects the returned data value, but does not affect what gets stored in the query cache.

Confirmed that the select function does not impact the query cache 👍

Reply:
Nice, thanks for checking.